### DISCUSSION TITLE
Is it possible to enable GCP Traces and GCP Logs and GCP Managed Prometheus for storage?

### DISCUSSION THREAD

**Discussion by nikolaydubina** [SigNoz/signoz#5905] [Q&A] [answered]
I like signoz UI. but I do not want to run clickhouse or other databases for logs or traces or metrics.

How good is signoz support for those solutions? possible to deply just UI? (perhaps like single-pod/single-container of Grafana?)

**Comment by srikanthccv** [comment, marked as answer]
How good is signoz support for those solutions?

None. Everything built and being built is tightly coupled with ClickHouse. It should be possible to achieve the same by implementing the Reader interface for another DB or storage but such a thing doesn't exist.
